README.md

```diff
@@ -52,6 +52,12 @@ The CPC exercises autonomy in managing its responsibilities and seeks agreement
 * RequireJS
 * Sizzle
 
+## CPC Meetings
+
+The CPC [meets weekly](https://github.com/openjs-foundation/cross-project-council/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+in%3Atitle+Project+Council+Meeting+).  We publish meeting agendas as [issues](https://github.com/openjs-foundation/cross-project-council/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+in%3Atitle+Project+Council+Meeting+), and also [publish meetings on our calendar](https://calendar.google.com/calendar/embed?src=linuxfoundation.org_fuop4ufv766f9avc517ujs4i0g%40group.calendar.google.com). ([iCal](https://calendar.google.com/calendar/ical/linuxfoundation.org_fuop4ufv766f9avc517ujs4i0g%40group.calendar.google.com/public/basic.ics)) ([Add to Google Calendar](https://calendar.google.com/calendar?cid=bGludXhmb3VuZGF0aW9uLm9yZ19mdW9wNHVmdjc2NmY5YXZjNTE3dWpzNGkwZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t))
+
+CPC members should attend as many meetings as possible, and non-members are welcome to join as observers.  To add an item to the agenda, [create an issue and add the cross-project-council-agenda label](https://github.com/openjs-foundation/cross-project-council/issues?q=is%3Aissue+is%3Aopen+label%3Across-project-council-agenda).  
```

Comment:
Should we be explicit the non-member observers should be active in OpenJS projects?

Reply:
I'm flexible on the wording, and will defer to others on this.

Reply:
I think we should be explicit.

Reply:
let's be explicit.

Reply:
Since Observer is a term that comes from the language of the Charter, how about we just link to it? It's the definition that would control, here... 

https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md#observers

Reply:
This is no longer relevant based on last weeks conversation